flink-end-to-end-tests/test-scripts/docker-hadoop-secure-cluster/bootstrap.sh

```diff
@@ -92,7 +92,12 @@ if [ "$1" == "--help" -o "$1" == "-h" ]; then
     exit 0
 elif [ "$1" == "master" ]; then
     yes| sudo -E -u hdfs $HADOOP_PREFIX/bin/hdfs namenode -format
-
+    sudo -E sudo netstat -tulpn >> /var/log/hadoop/debugging.out
```

Comment:
Just as a side-note: I usually added also `-e` as a parameter getting "extended" output. Not sure whether it helps in that case but it's just nice to read for someone who knows German: `netstat -tulpen` 😄

Reply:
I tried the -e argument, but it only adds things like "package count" per port, which is not relevant for understanding who's allocating the port. (I copy pasted "tulpn" from somewhere, and also remembered the -e)